requirements.txt

```diff
@@ -32,3 +32,6 @@ bleach
 bleach-whitelist
 Pillow
 beautifulsoup4
+rq
+rq-dashboard
```

Comment:
Let's keep this optional, because unless a firewall is added, this could expose running jobs and its data in a production environment, unless it has authentication configured